eng/ProjectReferences.props

```diff
@@ -56,7 +56,6 @@
     <ProjectReferenceProvider Include="Microsoft.Extensions.Options.DataAnnotations" ProjectPath="$(RepoRoot)src\Options\DataAnnotations\src\Microsoft.Extensions.Options.DataAnnotations.csproj" RefProjectPath="$(RepoRoot)src\Options\DataAnnotations\ref\Microsoft.Extensions.Options.DataAnnotations.csproj" />
     <ProjectReferenceProvider Include="Microsoft.Extensions.Options" ProjectPath="$(RepoRoot)src\Options\Options\src\Microsoft.Extensions.Options.csproj" RefProjectPath="$(RepoRoot)src\Options\Options\ref\Microsoft.Extensions.Options.csproj" />
     <ProjectReferenceProvider Include="Microsoft.Extensions.Primitives" ProjectPath="$(RepoRoot)src\Primitives\src\Microsoft.Extensions.Primitives.csproj" RefProjectPath="$(RepoRoot)src\Primitives\ref\Microsoft.Extensions.Primitives.csproj" />
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Testing" ProjectPath="$(RepoRoot)src\TestingUtils\Microsoft.AspNetCore.Testing\src\Microsoft.AspNetCore.Testing.csproj" RefProjectPath="$(RepoRoot)src\TestingUtils\Microsoft.AspNetCore.Testing\ref\Microsoft.AspNetCore.Testing.csproj" />
```

Comment:
I've lost some context. Why turn off `$(IsProjectReferenceProvider)` for Microsoft.AspNetCore.Texting? That is, why not just change `$(IsProjectReferenceProvider)`?

Reply:
I'm not quite understanding what you are saying here? I did change `IsProjectReferenceProvider` to false for this project.

Reply:
I fixed my topo after making the original comment. Think you're responding to the email notification rather than what's here. In any case, it seems unnecessary to change more than `$(HasReferenceAssembly)`.

Reply:
>In any case, please update Microsoft.AspNetCore.Testing to remove the ref/ project and change other projects to use an @(ProjectReference) item when they depend on Microsoft.AspNetCore.Testing.
To make it clear what's going on later, please also mark Microsoft.AspNetCore.Testing with <IsProjectReferenceProvider>false</IsProjectReferenceProvider>.

We wanted to ensure that any references to M.A.Testing to use `ProjectReference` and that a `Reference` to the project would fail.

Reply:
Thanks for the reminder.